__tests__/integration.test.js

```diff
@@ -196,6 +196,25 @@ describe("Integration tests", () => {
 						expect(articles).toBeSortedBy("created_at", { descending: true });
 					});
 			});
+			test("GET:200 endpoint should accept a topic query which filters result to only the specified topic", () => {
+				return request(app)
+					.get("/api/articles?topic=cats")
+					.expect(200)
+					.then(({ body: { articles } }) => {
+						expect(articles.length).toBe(1);
+						articles.forEach(({ topic }) => {
+							expect(topic).toBe("cats");
+						});
+					});
+			});
+			test("GET:200 sends an empty array when provided with a topic that doesn't have any articles", () => {
```

Comment:
I agree that a valid topic with no articles should be a 200. Does the topic dogs exist in this database? Else why would we have a table of valid topics to check 😄 If dogs does not exist in the data base, what status code should that be?